dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/WECC/Parameters.mo

```diff
@@ -105,4 +105,11 @@ record Parameters "Parameters of the PV WECC model"
   annotation(preferredView = "text");
   end Params_VSourceRef;
 
+  record Params_DriveTrain
+    parameter Types.Time Ht "Turbine Inertia in seconds (typical: 5s)";
+    parameter Types.Time Hg "Generator Inertia in seconds (typical: 1s)";
+    parameter Types.PerUnit Dshaft "Damping coefficient in p.u. (typical: 1.5 p.u.)";
+    parameter Types.PerUnit Kshaft "Spring constant in p.u. (typical: 200 p.u.)";
```

Comment:
base

Reply:
is Kshaft influenced by the base choice?

Reply:
Same here